### ISSUE TITLE
Calculate series atom

### ISSUE THREAD

**Issue by levBagryansky** [objectionary/eo-math#56]
What if we introduce a new atom that calculates a series. So we could implement `sin`, `cos` and and othe trigonometric functions via Taylor series. It would take a value and formula for coefficient of nth term. For example we could implement exp(x) by this:
```
[x] > exp
  calc-series > @
    x
    "1/n!"
```
We need to give coefficient of nth term somehow.
@Graur @EugeneDar @includealex what do you think?

**Comment by levBagryansky** [comment]
@Graur May be it would be better to implement this in eo. Precise woulbe worse but we could give a function like this:
```
[x] > exp
  calc-series > @
    x
    [n]
      div.
        1
        n
```

**Comment by Graur** [comment]
@levBagryansky Tailor series sounds good to me. But maybe we can read more about Chebyshev Polynomials first? Since this algorithm is used in Java and C libraries, as far as I know.
@yegor256 What do you think about implementation of trigonometric functions via Tailor series?

**Comment by yegor256** [comment]
@levBagryansky @Graur I like the idea, but I would call the atom `taylor`

**Comment by levBagryansky** [comment]
@Graur please assign me

**Comment by levBagryansky** [comment]
@Graur rultor did not open `@todo` issues

**Comment by 0pdd** [comment]
@levBagryansky 4 puzzles [#79](https://github.com/objectionary/eo-math/issues/79), [#81](https://github.com/objectionary/eo-math/issues/81), [#82](https://github.com/objectionary/eo-math/issues/82), [#83](https://github.com/objectionary/eo-math/issues/83) are still not solved.

**Comment by 0pdd** [comment]
@levBagryansky 3 puzzles [#79](https://github.com/objectionary/eo-math/issues/79), [#81](https://github.com/objectionary/eo-math/issues/81), [#82](https://github.com/objectionary/eo-math/issues/82) are still not solved; solved: [#83](https://github.com/objectionary/eo-math/issues/83).
